contracts/ronin/staking/Staking.sol

```diff
@@ -119,10 +119,12 @@ contract Staking is IStaking, StakingCallback, Initializable, AccessControlEnume
   function execDeductStakingAmount(
     address poolId,
     uint256 amount
-  ) external override onlyContract(ContractType.VALIDATOR) returns (uint256 actualDeductingAmount_) {
+  ) external override onlyContract(ContractType.VALIDATOR) returns (uint256 actualDeductingAmount_, bool success) {
     actualDeductingAmount_ = _deductStakingAmount(_poolDetail[poolId], amount);
     address payable validatorContractAddr = payable(msg.sender);
+    success = true;
     if (!_unsafeSendRON(validatorContractAddr, actualDeductingAmount_)) {
+      success = false;
```

Comment:
This is not correct. The validator here is slashed, but the deducting is failed. Return it as `not slashed` is confusing and will cause many improper handling.

Reply:
will revert this changes: confirmed by https://sky-mavis.slack.com/archives/C05DYV21A72/p1713951395869659?thread_ts=1713950652.985619&cid=C05DYV21A72